content/en/serverless/step_functions/installation.md

```diff
@@ -66,6 +63,127 @@ Then, to send your Step Functions logs to Datadog:
+
+   <div class="alert alert-info">If you enable tracing (which automatically includes enhanced metrics), you are billed for both Serverless Workload Monitoring and Serverless APM. See <a href="https://www.datadoghq.com/pricing/?product=serverless-monitoring#products">Pricing</a>.</div>
+
+[1]: /serverless/libraries_integrations/cli/
+[2]: /logs/guide/forwarder
+[3]: /logs/guide/send-aws-services-logs-with-the-datadog-kinesis-firehose-destination
+[4]: https://github.com/DataDog/datadog-ci/blob/master/src/commands/stepfunctions/README.md
+{{% /tab %}}
+{{% tab "Serverless Plugin" %}}
+
+1. If you have not already, install the [Datadog Serverless Framework Plugin][1] v5.40.0+:
+
+    ```shell
+    serverless plugin install --name serverless-plugin-datadog
+    ```
+
+1. Ensure you have deployed the [Datadog Lambda Forwarder][2], and that you are using v3.130.0+. You might need to [update your Forwarder][2]. As an alternative, you may also use [Amazon Data Firehose][3], which can subscribe to Amazon CloudWatch log groups across multiple AWS regions. Your Step Functions log group name should use this format: 
+   ```
+   /aws/vendedlogs/states/<STEP_FUNCTION_NAME>-Logs-<STAGE>
+   ```
+
+1. Add the following to your `serverless.yml`:
+
+   ```yaml
+   custom:
+     datadog:
+       site: <DATADOG_SITE>
+       apiKeySecretArn: <DATADOG_API_KEY_SECRET_ARN>
+       forwarderArn: <FORWARDER_ARN>
+       enableStepFunctionsTracing: true
+       propagateUpstreamTrace: true
+       mergeStepFunctionAndLambdaTraces: true
+   ```
+
+    - Replace `<DATADOG_SITE>` with {{< region-param key="dd_site" code="true" >}} (ensure the correct SITE is selected on the right).
+    - Replace `<DATADOG_API_KEY_SECRET_ARN>` with the ARN of the AWS secret where your [Datadog API key][4] is securely stored. The key needs to be stored as a plaintext string (not a JSON blob). The `secretsmanager:GetSecretValue` permission is required. For quick testing, you can instead use `apiKey` and set the Datadog API key in plaintext.
+    - Replace `<FORWARDER_ARN>` with the ARN of your Datadog Lambda Forwarder. This step configures the log stream subscription for the Forwarder. 
+
+    For additional settings, see [Datadog Serverless Framework Plugin - Configuration parameters][5].
+
+1. Set up tags. Open your AWS console and go to your Step Functions state machine. Open the *Tags* section and add the following key-value pairs:
+   | key | value |
+   | --  | ----- |
+   | `env` | Name of your environment. Required to see traces in Datadog. Defaults to `dev`. |
+   | `service` | Name of your service. Defaults to the state machine's name. |
+   | `dd_version` | `dd_sls_ci`| 
+
+1. To enable tracing, you have two options:
+   - **Per Step Function**: Add the `DD_TRACE_ENABLED` tag to each Step Function and set the value to `true`.
```

Comment:
We only need to add `enableStepFunctionsTracing: true`, as shown in the example code above.